src/app/AppConfigContext.tsx

```diff
@@ -28,6 +31,9 @@ const AppConfigContext = createContext<AppConfig>({
       // The default value is 40 MB. The threshold must be set to at least 1 MB, and can be set to CACHE_SIZE_UNLIMITED to disable garbage collection.
     },
   },
+  municipality: {
+    name: "Test deployment",
+  },
 });
```

Comment:
Can you move the footer links in here too? So that we have a single file for the maintainer to edit to customize their instance?

Reply:
Is that even possible?

Reply:
That's definitely possible.

Would have to just modify `AppConfig` type to have a list of links field and then provide those links to `AppConfigProvider`.

Something like: 
```
interface Link {
  label: string
  url: string
}
municipality: {
  footerSecondaryLinks: Link[]
}
```

Reply:
Works for me!